minecraft_servers/survivalgames/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "server_name": "survivalgames",
+  "nice_name": "SurvivalGames",
+  "direct_ip": "mc.survival-games.cz",
+  "supported_languages": [
+     "cs",
+     "sk"
+	 "en"
+  ],
+  "social": {
+    "web": "https://survival-games.cz",
+    "web_shop": "https://store.survival-games.cz",
+    "web_support": "https://account.survival-games.cz",
+    "facebook": "CZSurvivalGames", 
+    "instagram": "survival.games.cz",
+    "youtube": "https://www.youtube.com/@SurvivalGamesCZE"
+	"discord": "https://discord.com/invite/ZvK9g2WaVs"
+	"teamspeak": "ts3.survival-games.cz"
+	"tiktok": "https://www.tiktok.com/@survivalgamescz"
+  },
+  "discord": {
+    "server_id": 694511728691511347,
+    "rename_to_minecraft_name": true
```

Comment:
Reserved for partners only; change to `false`

Reply:
I changed it now